payments/serializers/v0/__init__.py

```diff
@@ -109,3 +154,91 @@ class Meta:
             "total_price",
         ]
         model = Basket
+
+
+class LineSerializer(serializers.ModelSerializer):
+    """Serializes a line item for an order."""
+
+    product = ProductSerializer()
+    unit_price = serializers.SerializerMethodField()
+    total_price = serializers.SerializerMethodField()
+
+    def get_unit_price(self, instance):
+        """Get the unit price for the line."""
+        return str(instance.unit_price.quantize(TWO_DECIMAL_PLACES))
+
+    def get_total_price(self, instance):
+        """Get the total price for the line."""
+        return str(instance.total_price.quantize(TWO_DECIMAL_PLACES))
+
+    class Meta:
+        """Meta options for LineSerializer"""
+
+        fields = [
+            "id",
+            "quantity",
+            "item_description",
+            "unit_price",
+            "total_price",
+            "product",
+        ]
+        model = Line
+
+
+class WebhookBaseSerializer(serializers.Serializer):
+    """Base serializer for webhooks."""
+
+    system_key = serializers.CharField()
+    type = serializers.SerializerMethodField()
+    user = UserSerializer()
+
+    def get_type(self, instance):
```

Comment:
It would probably be a bit cleaner to do this as `type=Charfield(choices=PAYMENT_WEBHOOK_CHOICES)` and then later when you go to send the webhook do something like this:

```python

if serializer.is_valid():
     requests.post(url, serializer.data)
else:
     log.error("Errors: %s", serializer.errors)